Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture_test.py

```diff
@@ -111,6 +113,51 @@ def test_get_indicators_command(mocker, indicator_type, build_iterator_answer, v
     assert entry_result[0]['Type'] == type
 
 
+GET_INDICATORS_BY_RISK_RULES_INPUTS = [
+    ('url', 'dhsAis', build_iterator_answer_url, 'www.securityadvisor.io', 'URL'),
+    ('url', 'dhsAis,phishingUrl', build_iterator_answer_url, 'www.securityadvisor.io', 'URL'),
+    ('ip', 'dhsAis,phishingUrl,defangedURL', build_iterator_answer_ip, '192.168.1.1', 'IP')
+]
+
+
+@pytest.mark.parametrize('indicator_type, risk_rules, build_iterator_answer, value, type',
+                         GET_INDICATORS_BY_RISK_RULES_INPUTS)
+def test_get_indicators_command_by_risk_rules(mocker, indicator_type, risk_rules, build_iterator_answer, value, type):
+    """
+    Given:
+     - Recorded Future Feed client initialized with a 'ConnectApi' service, and a:
+      1. URL indicator type, and a valid risk rule.
+      2. URL indicator type, and a comma separated list of two valid risk rules.
+      3. IP indicator type, and a comma separated list of three valid risk rules.
+
+     - Mock response of the fetched indicators
+
+    When:
+     - Running the 'get_indicators_command'
+
+    Then:
+     - Verify the raw response and the human readable output of the command are correct, and include fetched indicators
+      of all the defined risk rules.
+    """
+    from FeedRecordedFuture import get_indicators_command, Client
+    client = Client(indicator_type=indicator_type, api_token='123', risk_rule=risk_rules, services='ConnectApi')
+    args = {
+        'indicator_type': indicator_type,
+        'limit': 1
+    }
+    mocker.patch('FeedRecordedFuture.Client.build_iterator')
```

Comment:
can we use `requests_mocks` instead of this ?

Reply:
> please add to content-test-conf another instance of the feed where there is comma-seperated list of risk rules

@Itay4  - added in this merge request: [1339](https://code.pan.run/xsoar/content-test-conf/-/merge_requests/1339)